pages/linux/lxterminal.md

```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open a lxterminal window:
```

Comment:
```suggestion
- Open an lxterminal window:
```

Reply:
I thought that as well however its not a vowel.

Reply:
No, it isn't.  But you're supposed to use `an` before if either (1) the first letter of the next word is a vowel, or (2) the word starts with a phonetic vowel. `lxterminal` phonetically spelled out is `elexterminal`, which does begin with a vowel 😉